src/main/java/com/flightstats/hub/spoke/SpokeWriteContentDao.java

```diff
@@ -89,14 +90,15 @@ public Content get(String channelName, ContentKey key) {
         String path = getPath(channel, limitKey);
         log.trace("latest {} {}", channel, path);
         traces.add("SpokeWriteContentDao.latest", channel, path);
+        Optional<ContentKey> key;
```

Comment:
Another case where you can safely decrease the scope of this variable. Though, it looks like you intentionally increased the scope?
```
try {
    Optional<ContentKey> key = chronoStore.getLatestFromCluster(channel, path, traces);
    traces.add("SpokeWriteContentDao.latest", key);
    return key;
} catch Exception (e) {
    log.warn("what happened? " + channel, e);
    return Optional.empty();
}
```

Reply:
The flow wasn't explicit with that catch-all return that could only actually be hit if there was an exception.  Returning directly is easier to read here; will switch to that.